src/helper.py

```diff
@@ -198,3 +241,18 @@ def validate_inputs():
 
     validate_inputs()
     return get_assets_fullname_from_collibra()
+
+
+def get_asset_types_name_from_lineage_json_file(path: str) -> set:
+    types = set([])
+    with open(path) as f:
```

Comment:
should we check if file exist?

Reply:
I did not want to add a try/expect for that, so the user using the function can get the exception.
Do you think we still need a try/expect based on that?